README.md

```diff
@@ -16,4 +16,7 @@ AccessibleWebDev is licensed under the MIT License - see [License](https://githu
 
 ## Gitpod
 
+Gitpod ia an open-source Kubernetes application which provides prebuilt, collaborative and development environment in your browser.  
+Gitpod allows you to view and interact with this code directly in your browser without the need to clone it to your local machine. If you wish to make changes that lead to a pull request, remember to fork the repository then click the Gitpod button in the forked readme. Otherwise, feel free to play around with the code by clicking here.
```

Comment:
```suggestion
It allows you to view and interact with this code directly in your browser without the need to clone it to your local machine. If you wish to make changes that lead to a pull request, remember to fork the repository then click the Gitpod button in the forked readme. Otherwise, feel free to play around with the code by clicking here.
```

Reply:
Yes, I understood the changes. I'll do it.